contrib/vagrant/start.sh

```diff
@@ -51,6 +51,9 @@ export 'CILIUM_SCRIPT'=true
 # Sets the directory where the temporary setup scripts are created
 export 'CILIUM_TEMP'="${dir}"
 
+# Sets VM's Command wget with HTTPS_PROXY
+export 'VM_PROXY'="${VM_SET_PROXY}"
```

Comment:
This uses `VM_SET_PROXY`, but I don't see that anywhere else in this PR, including in the documentation. Where does it come from?